frontend/app-development/router/PageRoutes.tsx

```diff
@@ -1,30 +1,40 @@
 import React from 'react';
-import classes from './PageRoutes.module.css';
-import { Navigate, Route, Routes } from 'react-router-dom';
+import {
+  RouterProvider,
+  createBrowserRouter,
+  createRoutesFromElements,
+  Navigate,
+  Route,
+} from 'react-router-dom';
 import { AppShell } from 'app-development/layout/AppShell';
+import { PageLayout } from 'app-development/layout/PageLayout';
 import { RoutePaths } from 'app-development/enums/RoutePaths';
 import { routerRoutes } from 'app-development/router/routes';
 import { NotFoundPage } from 'app-shared/components/notFound';
+import { APP_DEVELOPMENT_BASENAME } from 'app-shared/constants';
 
 const BASE_PATH = '/:org/:app';
 
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path='/' element={<AppShell />}>
```

Comment:
Should we rename `<AppShell>` back to just `App.tsx`? I think this was separated before because we needed to abstract the code into two layers. Within this PR it looks like `App.tsx` and `AppShell.tsx` have been merged. 

Then I suggest renaming `AppShell` to `App` to stay consistent with other packages and how default React projects are created.